knowledge-base/images-disappear-from-word-report-after-save.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Images Disappear from the DOCX Report Document after Saving It in Word
+description: In large Word documents images may disappear after saving
+type: troubleshooting
+page_title: Images not shown in Word report
+slug: images-disappear-from-word-report-after-save
+position: 
+tags: 
+ticketid: 1536895
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
```

Comment:
You may add a row to the environment table stating that the rendering format/extension is DOCX.